app/services/katello/pulp3/api/content_guard.rb

```diff
@@ -0,0 +1,45 @@
+require 'pulp_certguard_client'
+
+module Katello
+  module Pulp3
+    module Api
+      class ContentGuard < Core
+        def self.client_module
+          PulpCertguardClient
+        end
+
+        def self.api_exception_class
+          PulpCertguardClient::ApiError
+        end
+
+        def api_client
+          PulpCertguardClient::ApiClient.new(smart_proxy.pulp3_configuration(PulpCertguardClient::Configuration))
+        end
+
+        def rhsm_api_client
+          PulpCertguardClient::ContentguardsRhsmApi.new(api_client)
+        end
+
+        def create(name = "RHSMCertGuard")
```

Comment:
Perhaps ca_certificate should be an argument here - with the default being `Cert::Certs.ca_cert`

Reply:
Good point..will update :+1: